Makefile

```diff
@@ -21,36 +21,44 @@
- WORK_ROOT = ${HOME}/coriolis-2.x/Linux.el9/Release.Shared
+ ISYS_ROOT    = $(shell ./bootstrap/coriolisEnv.py --query-inst-root)
+ SRC_ROOT     = $(shell ./bootstrap/coriolisEnv.py --query-src-root)
+ BUILD_ROOT   = $(ISYS_ROOT)/build
+ INST_ROOT    = $(ISYS_ROOT)/install
+ SRC_CORIOLIS = $(SRC_ROOT)/coriolis
+ SRC_ALLIANCE = $(SRC_ROOT)/alliance/alliance/src
 
 
 help:
 	@echo "============================================================================"; \
 	 echo "Coriolis build & install top Makefile";                                        \
 	 echo "";                                                                             \
-	 echo "This Makefile is only a wrapper around the Coriolis & Chams Builder tool";     \
-	 echo "(./boostrap/ccb.py)";                                                          \
+	 echo "This Makefile is only a wrapper around virtualenv, pdm & meson.";              \
 	 echo "";                                                                             \
 	 echo "To build it, simply type (in coriolis/, at the shell prompt):";                \
 	 echo "    ego@home:coriolis> make install";                                          \
 	 echo "To remove the build directory (and keep the software installed):";             \
 	 echo "    ego@home:coriolis> make clean_build";                                      \
 	 echo "To fully remove build & install:";                                             \
 	 echo "    ego@home:coriolis> make uninstall";                                        \
+	 echo "To build Alliance";                                                            \
+	 echo "    ego@home:coriolis> make install_alliance";                                 \
+	 echo "    ";                                                                         \
+	 echo "    Note: The Alliance repository must have been cloned beforehand";           \
 	 echo "============================================================================"; \
-	 echo "SMP_FLAGS = $(SMP_FLAGS)" ;                                                    \
-	 echo "ISYS_ROOT = $(ISYS_ROOT)" ;                                                    \
-	 echo "INST_ROOT = $(INST_ROOT)" ;                                                    \
+	 echo "SMP_FLAGS    = $(SMP_FLAGS)" ;                                                 \
+	 echo "BUILD_ROOT   = $(BUILD_ROOT)" ;                                                \
+	 echo "INST_ROOT    = $(INST_ROOT)" ;                                                 \
+	 echo "SRC_CORIOLIS = $(SRC_CORIOLIS)" ;                                              \
+	 echo "SRC_ALLIANCE = $(SRC_ALLIANCE)" ;                                              \
 	 echo "============================================================================";
 
 
 check_dir:
-	@if [ "`pwd`" != "${HOME}/coriolis-2.x/src/coriolis" ]; then      \
+	@if [ "`pwd`" != "$(SRC_CORIOLIS)" ]; then                        \
```

Comment:
This logic shouldnt be needed any more, i think?

Reply:
If you use the ``Makefile``, I assume you are a developer working in your home directory, so it still make sense to me.

Reply:
this is a weird development environment. Again, see comment on confusing for incoming users and relevancy only for existing LIP6 users.